### PR TITLE
Updated react modals to mantine

### PR DESCRIPTION
**Pull request by Jin1L**
- Updated to mantine style
- format

### CODE REVIEW

**Review by itsjoeoui — APPROVED** [review]
lgtm

File: src/feature/flow_chart_panel/views/NodeModal.tsx [hunk 1]
```diff
@@ -1,9 +1,10 @@
-import ReactModal from "react-modal";
 import SyntaxHighlighter from "react-syntax-highlighter";
 import { docco, srcery } from "react-syntax-highlighter/dist/cjs/styles/hljs";
 import PlotlyComponent from "../../common/PlotlyComponent";
+import { Modal, useMantineTheme } from "@mantine/core";
 import { createStyles, useMantineColorScheme } from "@mantine/styles";
 import { NodeModalProps } from "../types/NodeModalProps";
+import { useState, useEffect } from "react";
```
Comment (smahmed776) [inline src/feature/flow_chart_panel/views/NodeModal.tsx:7]: I think this import is unused.

**Review by smahmed776 — CHANGES_REQUESTED** [review]
Looks good, some cosmetic changes were requested.

File: src/__tests__/feature/flow_chart_panel/views/NodeModal.test.tsx [hunk 2]
```diff
@@ -33,14 +33,14 @@ jest.mock("@src/feature/common/PlotlyComponent", () => {
...
 
-jest.mock("react-modal", () => {
-  const ReactModal = jest
-    .fn()
-    .mockReturnValue(<div data-testid="react-modal" />);
-  return {
-    default: ReactModal,
-  };
-});
+// jest.mock("@mantine/core", () => {
```
Comment (smahmed776) [inline src/__tests__/feature/flow_chart_panel/views/NodeModal.test.tsx:36]: Can we just get rid of unnecessary comments, Please?

File: src/__tests__/feature/flow_chart_panel/views/NodeModal.test.tsx [hunk 3]
```diff
@@ -6,7 +6,7 @@ const props: NodeModalProps = {
   modalIsOpen: false,
   afterOpenModal: jest.fn(),
   closeModal: jest.fn(),
-  modalStyles: {},
+  //modalStyles: {},
```
Comment (smahmed776) [inline src/__tests__/feature/flow_chart_panel/views/NodeModal.test.tsx:9]: I think we shouldn't keep any unnecessary comments in code.

**Review by smahmed776 — CHANGES_REQUESTED** [review]
Looks good, I recommend removing unnecessary comments.

**Review by smahmed776 — APPROVED** [review]
LGTM

**Pull request merged**